src/routers/alpha-router/alpha-router.ts

```diff
@@ -1001,9 +1001,17 @@ export class AlphaRouter
       const quoteDiff = swapRouteFromChain.quote.subtract(swapRouteFromCache.quote);
       const quoteGasAdjustedDiff = swapRouteFromChain.quoteGasAdjusted.subtract(swapRouteFromCache.quoteGasAdjusted);
       const gasUsedDiff = swapRouteFromChain.estimatedGasUsed.sub(swapRouteFromCache.estimatedGasUsed);
+      const gasAdjustedDiffInAmountTokenTerms = quoteGasAdjustedDiff.multiply(amount.divide(swapRouteFromChain.quoteGasAdjusted));
```

Comment:
if this diff is in "tokenTerms", which terms is quoteGasAdjustedDiff in? Is it not a token?? I'm finding the naming a bit hard to follow